aspnet/tutorials/first-mvc-app/adding-view.rst

```diff
@@ -48,25 +48,25 @@ Changing views and layout pages
 
 Tap on the menu links (**MvcMovie**, **Home**, **About**). Each page shows the same menu layout. The menu layout is implemented in the *Views/Shared/_Layout.cshtml* file. Open the *Views/Shared/_Layout.cshtml* file.
 
-Layout templates allow you to specify the HTML container layout of your site in one place and then apply it across multiple pages in your site. Find the ``@RenderBody()`` line. ``RenderBody`` is a placeholder where all the view-specific pages you create show up, "wrapped" in the layout page. For example, if you select the **About** link, the **Views/Home/About.cshtml** view is rendered inside the ``RenderBody`` method.
+:doc:`/mvc/views/layout` templates allow you to specify the HTML container layout of your site in one place and then apply it across multiple pages in your site. Find the ``@RenderBody()`` line. ``RenderBody`` is a placeholder where all the view-specific pages you create show up, "wrapped" in the layout page. For example, if you select the **About** link, the **Views/Home/About.cshtml** view is rendered inside the ``RenderBody`` method.
```

Comment:
>I think I would explicitly specify the text for this link in case the title of the linked doc changes.
 
@danroth27  That's a good policy. I added it to https://github.com/aspnet/Docs/issues/1054 style guide issues